streams/my.m3u

```diff
@@ -47,21 +47,21 @@ https://ythls.onrender.com/channel/UCRzAqsmluZYk-r59pQYSa3Q.m3u8
 https://ythls.onrender.com/channel/UCt4ZHf2zhAzkq3iI5tG0yFg.m3u8
 #EXTINF:-1 tvg-id="",ERA (720p) [Not 24/7]
 https://ythls.onrender.com/channel/UCNq5APn4lVabRaYCnTxiNIg.m3u8
-#EXTINF:-1 tvg-id="",TV Selangor (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="",TV Selangor (720p) [Not 24/7] [Geo-blocked]
 https://ythls.onrender.com/channel/UCldPEJgJgq_Mb6Ndek5RxQg.m3u8
-#EXTINF:-1 tvg-id="",RTM Parlimen (Dewan Rakyat)
+#EXTINF:-1 tvg-id="",RTM Parlimen (Dewan Rakyat) [Geo-blocked]
 https://d25tgymtnqzu8s.cloudfront.net/smil:rakyat/playlist.m3u8?id=7
-#EXTINF:-1 tvg-id="",RTM Parlimen (Dewan Negara)
+#EXTINF:-1 tvg-id="",RTM Parlimen (Dewan Negara) [Geo-blocked]
```

Comment:
For these hosts:

Add 
#EXTVLCOPT:http-referrer=https://rtm-player.glueapi.io/

Reply:
curieusement je lai joues sans ref et sont pas geo ici